hvpm.py

```diff
@@ -48,7 +53,7 @@ def main(unused_argv):
 
   if FLAGS.status:
     print "Serialno : %d\n" % mon.getSerialNumber();
-    mon.fillAllStatusPacket()
```

Comment:
Leaving this open for the moment.  The intention of fillStatusPacket() is what you're suspecting it was, to fill in every applicable value from the firmware.  Since I removed fillAllStatusPacket() from HVPM when merging them, your code will be break slightly.